assets/js/vue/components/search/ResultSet.vue

```diff
@@ -89,15 +99,15 @@
                         queryParam: 'status'
                     },
                     fundingCycle: {
-                        label: 'Funding Cycles',
+                        label: fundingCycleAlias,
```

Comment:
This changed code in another pull request. Remove these here, and redo them in the other pull request. Also see noted in that pull request.